EternalModLoader/EternalModLoader.cs

```diff
@@ -249,19 +250,112 @@ public static void ReplaceChunks(MemoryStream memoryStream, ResourceInfo resourc
                     long sizeDiff = mod.FileBytes.Length - size;
 
+                    // If the mod is a blang file json, modify the .blang file
+                    if (mod.isBlangJson)
+                    {
+                        BlangJson blangJson;
+                        try
+                        {
+                            blangJson = JsonConvert.DeserializeObject<BlangJson>(Encoding.UTF8.GetString(mod.FileBytes));
+                            if (blangJson == null || blangJson.strings.Count == 0)
+                                throw new Exception();
+                            foreach (var blangJsonString in blangJson.strings)
+                            {
+                                if (blangJsonString == null || blangJsonString.name == null || blangJsonString.text == null)
+                                    throw new Exception();
+                            }
+                        }
+                        catch
+                        {
+                            Console.ForegroundColor = ConsoleColor.Red;
+                            Console.Write("ERROR: ");
+                            Console.ResetColor();
+                            Console.WriteLine($"Failed to parse EternalMod/strings/{Path.GetFileNameWithoutExtension(mod.Name)}.json");
+                            continue;
+                        }
+                        
+                        memoryStream.Seek(fileOffset, SeekOrigin.Begin);
+                        byte[] blangFileBytes = new byte[size];
+                        memoryStream.Read(blangFileBytes, 0, (int)size);
+                        int res = BlangCrypt.IdCrypt(ref blangFileBytes, $"strings/{Path.GetFileName(mod.Name)}", true);
+
+                        if (res != 0)
+                        {
+                            continue;
+                        }
+
+                        BlangFile blangFile;
+                        using (var blangMemoryStream = new MemoryStream(blangFileBytes))
+                        {
+                            blangFile = BlangFile.Parse(blangMemoryStream);
+
+                        }
+
+                        foreach (var blangJsonString in blangJson.strings)
+                        {
+                            bool stringFound = false;
+                            
+                            foreach (var blangString in blangFile.Strings)
+                            {
+                                if (blangJsonString.name.Equals(blangString.Identifier))
+                                {
+                                    stringFound = true;
+                                    blangString.Text = blangJsonString.text;
+                                    Console.WriteLine($"\tReplaced {blangString.Identifier} in {mod.Name}");
+                                    break;
+                                }
+                            }
+
+                            if (stringFound)
```

Comment:
Always use braces, even for 1 line conditions. Please follow the C# coding convention: https://docs.microsoft.com/es-es/dotnet/csharp/programming-guide/inside-a-program/coding-conventions